src/compiler/crystal/compiler.cr

```diff
@@ -712,6 +730,12 @@ module Crystal
             builder.use_inliner_with_threshold = 150
           in .o0?
             # default behaviour, no optimizations
+          in .os?
+            builder.opt_level = 2
+            builder.use_inliner_with_threshold = 50
+          in .oz?
+            builder.opt_level = 2
+            builder.use_inliner_with_threshold = 5
           end
 
           builder.size_level = 0
```

Comment:
probably this size_level need to move inside case, and change for os,oz?

Reply:
Oh, thanks for pointing that out. That should be the final thing to do.